src/performance-metrics/index.json

```diff
@@ -0,0 +1,23 @@
+[
```

Comment:
I know `.includes` is much prettier, but if you make it an object, you'll save iterating this array during reduce